docs/section23/Rule23-19.md

```diff
@@ -0,0 +1,52 @@
+
+# Airside System - Rule 23-19  
+
+**Rule ID:** 23-19  
+**Rule Description:** Systems 5 - 8, the baseline system shall be modeled with preheat coils controlled to a fixed set point 20F less than the design room heating temperature setpoint.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**Appendix G Section Reference:** Section G3.1.3.19 Preheat Coils (Systems 5 through 8)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-5, 6, 7, 8  
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:**  
+
+1. baseline_hvac_type()  
+
+## Rule Logic:  
+
+- For each zone in B_RMR: `for zone_b in B_RMR...zones:`
+
+  - For each terminal in zone: `for terminal_b in zone_b.terminals:`
+
+    - Get HVAC system serving terminal: `hvac_b = terminal_b.served_by_heating_ventilation_air_conditioning_systems`
+  
+      - Check if HVAC system is type 5 through 8: `if baseline_hvac_type(hvac_b) in ["SYS-5", "SYS-6", "SYS-5", "SYS-8"]:`
+
+        - Set applicability flag: `rule_applicability_check = TRUE`
+
```

Comment:
Do Building->BuildingSegment->HVAC system instead, similar to 23-10?

Reply:
For this rule, we also need to get zone setpoint so it starts with zone.terminal.